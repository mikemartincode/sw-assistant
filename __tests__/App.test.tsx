import App from "../src/containers/App";
import React from "react";
import renderer from "react-test-renderer";

describe("Component: App", () => {
  it("should match its empty snapshot", () => {
    const tree = renderer.create(<App />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
