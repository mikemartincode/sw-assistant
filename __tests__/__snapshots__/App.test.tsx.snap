// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: App should match its empty snapshot 1`] = `
<div
  className="app"
>
  <h1>
    Hello World!
  </h1>
  <p>
    Foo to the barz
  </p>
</div>
`;
